src/components/Common/Layout/TaskColumn.tsx

```diff
@@ -0,0 +1,65 @@
+import { useQuery } from '@apollo/client';
```

Comment:
Column was replaced with TaskColumn, this component receives an status and makes the query with that value.
Each column is responsible of its own set of tasks, they get stored in a context (see TaskContext)